Stockwatch/Stockwatch.Business/Stockdataservices.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Stockwatch.Model;
+
+namespace Stockwatch.Business
+{
+    public class Stockdataservices
```

Comment:
Use pascal naming convention in C# StockDataServices

Reply:
Done